addons/mail/static/src/js/chatter.js

```diff
@@ -262,6 +275,45 @@ var Chatter = Widget.extend({
         });
 
     },
+    /**
+     * @private
+     *
+     * check if suggested receipient is already loaded then simply
+     * return _suggestedPartnersProm else first loads suggested receipient
+     * and then return _suggestedPartnersProm
+     */
+    _getSuggestedPartnersProm() {
```

Comment:
would rename it to `async _getSuggestedPartners()`, and would also update inner code to use es6 and async/await.
Example:
- `const result = await this._rpc();`
- `for (const recipient of threadRecipients)`